jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractMultiGridViewTest.java

```diff
@@ -199,4 +215,19 @@ public void testGlobalPreferences() {
         getView().loadListTable("key",
                                 consumer);
     }
+
+    @Test
+    public void testControlBulkOperationsEnabled() {
+
+        boolean enabled = false;
+        List a = Collections.singletonList(mockButton);
```

Comment:
@nmirasch one more thing, I would change the List from raw type to List<Button>, and move all these three lines into setupMocks methods.